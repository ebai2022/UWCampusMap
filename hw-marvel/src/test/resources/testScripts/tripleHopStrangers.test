# Testing different hops to get from different nodes to each other

LoadGraph g strangers.csv

# 3 hops
FindPath g Ethan Yufei

# 1 hop
FindPath g Ethan Elaine