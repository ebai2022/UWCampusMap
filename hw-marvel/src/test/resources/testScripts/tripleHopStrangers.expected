# Testing different hops to get from different nodes to each other

loaded graph g

# 3 hops
path from Ethan to Yufei:
Ethan to Elaine via CSE190B
Elaine to Andrew via MoveIn
Andrew to Yufei via CartPushing

# 1 hop
path from Ethan to Elaine:
Ethan to Elaine via CSE190B